person.rb

```diff
@@ -1,15 +1,16 @@
-require './corrector.rb'
+require './corrector'
 
 class Person
   attr_accessor :name, :age, :parent_permission
-  attr_reader :id, :corrector
+  attr_reader :id, :corrector, :rentals
 
-  def initialize(age, name = 'unknown', parent_permission: true)
+  def initialize(age, name = 'unknown', parent_permission = { parent_permission: true })
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
 
```

Comment:
- [ ] A method for adding rentals should be present. Link to the example in a previous lesson: [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship). Please do so to comply with the project requirements.

Reply:
Thank you for your valuable comment. I added the add_rental method as you can see in the image below:
![rentalinpersonmethod](https://user-images.githubusercontent.com/60292761/141265947-d749f97c-c918-41ba-93c6-4b15a21d5d9b.PNG)